### PR TITLE
add nvidia-515

### PR DESCRIPTION
**Pull request by hoewweken**
Until the 525 problem gets solved, i want the most actual 515 line to be available.
Version 515.105.01

### CODE REVIEW

File: components/openindiana/nvidia-515/Makefile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# Copyright 2022, Andreas Wacknitz
+# Copyright 2023, Stephan Althaus
+#
+
+BUILD_STYLE = archive
+BUILD_BITS = NO_ARCH
+
+include ../../../make-rules/shared-macros.mk
+
+# This is latest NVIDIA Production Branch driver
```
Comment (mtelka) [inline components/openindiana/nvidia-515/Makefile]: This is not the latest NVIDIA Production Branch driver

**Review by mtelka — CHANGES_REQUESTED** [review]

**Comment by AndWac** [comment]
Can you add a comment to the PR how you have tested this, please?

**Comment by hoewweken** [comment]
This is not the latest production branch driver,
but the latest (most recently updated) driver just beyond.
I think of this as an intermediate 'solution' until the 525-problems are solved.

I published the driver, installed it and i am using this version now, as stated by dmesg|grep nvidia:

$ pkg list|grep -i nvidia
driver/graphics/nvidia-515 (userland)             515.105.1-2023.0.0.0       i--

$ dmesg|grep -i nvidia
Apr 30 09:43:02 dell7720 nvidia_modeset: [ID 107833 kern.notice] Loading NVIDIA Kernel Mode Setting Driver for UNIX platforms  515.105.01  Mon Feb 27 12:42:09 UTC 2023
Apr 30 09:43:02 dell7720 pcieb: [ID 586369 kern.info] PCIE-device: display@0, nvidia1
Apr 30 09:43:02 dell7720 npe: [ID 236367 kern.info] PCI Express-device: display@0, nvidia1
Apr 30 09:43:02 dell7720 genunix: [ID 936769 kern.info] nvidia1 is /pci@0,0/pci8086,1901@1/display@0

**Comment by mtelka** [comment]
> This is not the latest production branch driver, but the latest (most recently updated) driver just beyond. I think of this as an intermediate 'solution' until the 525-problems are solved.

Yes, sure.  But the comment says it is latest.  Please update the comment.

Also, please note that the 515 branch will be [EOLed in May 2023](https://docs.nvidia.com/datacenter/tesla/drivers/#cuda-drivers).

**Comment by hoewweken** [comment]
... the merge of the latest 2 commits - i need to make a personal note how to achieve that :-) 
`git rebase -i HEAD~2` is the simpliest solution for these tiny things (?)

**Pull request merged**